attendance/misc/makeDB.py

```diff
@@ -1,61 +0,0 @@
-"""
-the script adds all data from the .npy files to the DB
-"""
-
-
-import pymongo
-import numpy as np
-import json
-
-"""
-writes list of question and answer from .npys to DB
-"""
-
-conn = pymongo.MongoClient('mongodb://chatbot-learn:qwerty1234@ds135592.mlab.com:35592/chatbot-learn')
```

Comment:
Also please look at the the failed travis build logs. they will show the places that are not linted.